<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSAudioCapture</class>
 <widget class="QWidget" name="KSAudioCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KSAudioCapture</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>输入:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxInput"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>输出:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxOutput"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonPlay">
     <property name="text">
      <string>开始播放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonPause">
     <property name="text">
      <string>暂停播放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonResume">
     <property name="text">
      <string>继续播放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonStop">
     <property name="text">
      <string>停止播放</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
